common/src/main/scala/uk/ac/wellcome/platform/archive/common/config/builders/HTTPServerBuilder.scala

```diff
@@ -8,9 +8,9 @@ import uk.ac.wellcome.typesafe.config.builders.EnrichConfig._
 
 object HTTPServerBuilder {
   def buildHTTPServerConfig(config: Config): HTTPServerConfig = {
-    val host = config.required[String]("http.host")
-    val port = config.required[Int]("http.port")
```

Comment:
The fact this has never thrown makes me suspicious.

Reply:
I don't like that I can't test this method either!